packages/hierarchies-react/src/presentation-hierarchies-react/internal/TreeModel.ts

```diff
@@ -151,6 +151,9 @@ export namespace TreeModel {
 
   export function setInstanceFilter(model: TreeModel, nodeId: string | undefined, filter?: GenericInstanceFilter): boolean {
     return updateForReload(model, nodeId, (node) => {
+      if (isTreeModelHierarchyNode(node)) {
```

Comment:
Do we want to expand collapsed nodes when filter is removed?

Reply:
no :)

Reply:
Fixed.